pinns/pinns.c

```diff
@@ -38,6 +52,8 @@ int main(int argc, char **argv) {
       {"cgroup", optional_argument, NULL, 'c'},
       {"dir", required_argument, NULL, 'd'},
       {"filename", required_argument, NULL, 'f'},
+      {"uid-mapping", optional_argument, NULL, UID_MAPPING},
```

Comment:
Hmm, reason why we use enum values here? Could it just be a long argument?

Reply:
The reason is that other options seem to follow the `unshare` tool flags.  And these two new options are not available there, so to not risk I've added only the long argument variant